Authy2FA/App_Start/IdentityConfig.cs

```diff
@@ -55,10 +55,10 @@ public static ApplicationUserManager Create(IdentityFactoryOptions<ApplicationUs
             manager.PasswordValidator = new PasswordValidator
             {
                 RequiredLength = 6,
-                RequireNonLetterOrDigit = true,
+                RequireNonLetterOrDigit = false,

```

Comment:
The original settings force to create a _complex_ password, I change this setting to ease the development process, as this is a demo application I consider that having a weak password is not a big deal, _but is just my opinion_.

If you have any concerns about this change I can restore the original values.